### PR TITLE
Paper and Documentation Cleaning

### PR DESCRIPTION
**Pull request by riccardocadei**
Address review https://github.com/openjournals/joss-reviews/issues/5587#issuecomment-1783945679

### CODE REVIEW

File: R/cre.R [hunk 1]
```diff
@@ -16,24 +16,24 @@
 #'     (default: 0.5).
 #'   - *Parameters for Discovery and Inference*
 #'     - *ite_method*: The method for ITE (pseudo-outcome) estimation
-#'     (default: 'aipw', options: {'aipw' for Augmented Inverse Probability
-#'     Weighting, 'cf' for Causal Forest, 'bart' for Causal Bayesian Additive
-#'     Regression Trees, slearner' for S-Learner, 'tlearner' for T-Learner,
-#'     'xlearner' for X-Learner, 'tpoisson' for T-Learner with Poisson
+#'     (default: "aipw", options: {"aipw" for Augmented Inverse Probability
```
Comment (Naeemkh) [inline R/cre.R:19]: Please drop the quotation marks as much as possible. If it is necessary, please use \dQuote. 
https://developer.r-project.org/Rds.html

File: R/generate_cre_dataset.R [hunk 2]
```diff
@@ -99,7 +97,7 @@ generate_cre_dataset <- function(n = 1000, rho = 0, n_rules = 2, p = 10,
 
   # Check for correct binary input
   if (!(binary_outcome %in% c(TRUE, FALSE))) {
-    stop("Invalid 'binary' input. Please specify TRUE or FALSE.")
+    stop("Invalid 'binary' input. Please specify `TRUE` or `FALSE`.")
```
Comment (Naeemkh) [inline R/generate_cre_dataset.R:100]: Please drop backticks from TRUE and FALSE in the code.

**Review by Naeemkh — CHANGES_REQUESTED** [review]
Thank you, Riccardo. Please consider the following guidelines:

- Minimize the use of quotations in the documentation. If quotations are inevitable, use double quotes and employ the \dQuote{a quotation} syntax.
- Maintain consistency throughout the documentation. Only R objects and functions mentioned should be enclosed in backticks.
- Include equations in the documentation using the \eqn command.
- Add two spaces after the end of a sentence in sections with running text, such as \description and \details.
For more information, please visit [the R documentation guidelines](https://developer.r-project.org/Rds.html)."

File: R/cre.R (hunk 1 above)
Comment (riccardocadei) [reply R/cre.R]: removed

File: R/generate_cre_dataset.R (hunk 2 above)
Comment (riccardocadei) [reply R/generate_cre_dataset.R]: Removed

**Comment by riccardocadei** [comment]
should I add 2 extra spaces at the end of each sentence in each description section?

**Pull request merged**